File: src/main/java/com/learn/guess/Game.java
package com.learn.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        int secret = random.nextInt(10)+1;
        System.out.println(secret);
        int number =0;
        while (number != secret){
            System.out.print("Please enter a number： ");
            number = scanner.nextInt();
            if (number > secret){
                System.out.println("lower");
            }else if (number < secret){
                System.out.println("higher");
            }else {
                System.out.println("Great! Number is " + secret);
            }
        }
        /*int i = 0;
        do {
            System.out.println(i);
            i++; -> i = i + 1;
        }while (i<3);*/
    }
}

File: src/main/kotlin/com/learn/Game.kt
package com.learn

import java.util.*
import kotlin.random.Random

fun main() {
//    val scanner = Scanner(System.`in`)
    val secret = Random.nextInt(10)+1
    println(secret)
    var number : Int = 0
    while (number != secret){
        print("Please enter number：")
        number = readLine()!!.toInt()
//        number = scanner.nextInt()
        if (number > secret){
            println("lower")
        }else if (number < secret){
            println("higher")
        }else{
            println("Great! Number is $secret")
        }
    }

}